packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/edit-name/component.js

```diff
@@ -0,0 +1,18 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  bindings: {
+    projectId: '<',
+    loadbalancer: '<',
```

Comment:
Not sure you need to inject loadbalancer here

Reply:
Yes indeed we need only projectId, region and poolId for the api call.